docs/lorawan-on-helium/frequency-plans/frequency-plans.mdx

```diff
@@ -206,7 +206,7 @@ Also known as **US902-928**
 | :------------ | :-------------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | Gabon         |                       |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
 | Gambia        |                       |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
-| Georgia       | unknown, limited CEPT | CEPT Rec. 70-03                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| Georgia       | EU863-870             | CEPT Rec. 70-03    http://mca.comcom.ge/spectrum_map/                                                                                                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
I can't independently confirm this based on your instructions. The page you linked doesn't load. 

I tried contacting the commission here: https://comcom.ge/en/contact

They use the same link on their site, so if nothing else maybe they need to fix it. :D

Reply:
Update, I've potentially confirmed the finding based on this resource: https://lora-alliance.org/wp-content/uploads/2020/11/RP_2-1.0.2.pdf

While checking PR (Lebanon): https://github.com/helium/docs/pull/386

Though if accurate, it should be add **EU433**:
```suggestion
| Georgia       | EU863-870   EU433  | [RP002-1.0.2 LoRaWAN® Regional Parameters (Nov. 2020)](https://lora-alliance.org/wp-content/uploads/2020/11/RP_2-1.0.2.pdf)                                                                                                                                                                                                                                                                                                                                                                                                                              |
```

Reply:
Thank you for your assistance and that Link from Lora Alliance, Time to talk with admins :)